src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_item.c

```diff
@@ -1,85 +1,1000 @@
+                            pauseCtx->cursorY[PAUSE_ITEM]--;
+                            pauseCtx->unk_298 = 4.0f;
+                            pauseCtx->cursorPoint[PAUSE_ITEM] -= 6;
+                            moveCursorResult = 1;
+                        } else {
+                            pauseCtx->cursorY[PAUSE_ITEM] = cursorY;
+                            pauseCtx->cursorPoint[PAUSE_ITEM] = cursorPoint;
+                        }
+                    } else if (pauseCtx->stickRelY < -30) {
+                        // move cursor down
+                        moveCursorResult = 2;
+                        if (pauseCtx->cursorY[PAUSE_ITEM] < 3) {
+                            pauseCtx->cursorY[PAUSE_ITEM]++;
+                            pauseCtx->unk_298 = 4.0f;
+                            pauseCtx->cursorPoint[PAUSE_ITEM] += 6;
+                            moveCursorResult = 1;
+                        } else {
+                            pauseCtx->cursorY[PAUSE_ITEM] = cursorY;
+                            pauseCtx->cursorPoint[PAUSE_ITEM] = cursorPoint;
+                        }
+                    }
+                }
+            }
+
+            cursorSlot = pauseCtx->cursorPoint[PAUSE_ITEM];
+            pauseCtx->cursorColorSet = 2;
+
+            if (moveCursorResult == 1) {
+                cursorItem = gSaveContext.save.inventory.items[pauseCtx->cursorPoint[PAUSE_ITEM]];
+            } else if (moveCursorResult != 2) {
+                cursorItem = gSaveContext.save.inventory.items[pauseCtx->cursorPoint[PAUSE_ITEM]];
+            }
+
+            if (cursorItem == ITEM_NONE) {
+                cursorItem = PAUSE_ITEM_NONE;
+                pauseCtx->cursorColorSet = 0;
+            }
+
+            if ((cursorItem != (u32)PAUSE_ITEM_NONE) && (msgCtx->msgLength == 0)) {
+                if (gSaveContext.buttonStatus[EQUIP_SLOT_A] == BTN_DISABLED) {
+                    gSaveContext.buttonStatus[EQUIP_SLOT_A] = BTN_ENABLED;
+                    gSaveContext.unk_3F22 = 0;
+                    Interface_ChangeAlpha(50);
+                }
+            } else if (gSaveContext.buttonStatus[EQUIP_SLOT_A] != BTN_DISABLED) {
+                gSaveContext.buttonStatus[EQUIP_SLOT_A] = BTN_DISABLED;
+                gSaveContext.unk_3F22 = 0;
+                Interface_ChangeAlpha(50);
+            }
+
+            pauseCtx->cursorItem[PAUSE_ITEM] = cursorItem;
+            pauseCtx->cursorSlot[PAUSE_ITEM] = cursorSlot;
+            if (cursorItem != PAUSE_ITEM_NONE) {
+                // Equip item to the C buttons
+                if ((pauseCtx->debugEditor == DEBUG_EDITOR_NONE) && !pauseCtx->itemDescriptionOn &&
+                    (pauseCtx->state == 6) && (pauseCtx->unk_200 == 0) &&
+                    CHECK_BTN_ANY(CONTROLLER1(&play->state)->press.button, BTN_CLEFT | BTN_CDOWN | BTN_CRIGHT)) {
+
+                    // Ensure that a transformation mask can not be unequipped while being used
```

Comment:
Is there a reason for this space?

Reply:
This section is filled with densely packed code and lots of macros, so the space here is just to help with readability

Reply:
Okay, wasn't sure if there was a particular logic to where it occurred. I agree it's useful but a lot of this is quite subjective.